ci/scripts/create-release.sh

```diff
@@ -18,6 +18,8 @@ mkdir -p release-candidate/certs
 rm -rf release-candidate/certs/*
 
 tar xvfz certs.s3/current-certificates.tgz -C release-candidate/certs
+test -d apps-certs.s3   && cp apps-certs.s3/*   release-candidate/certs/ || true
```

Comment:
It should stop the script from failing if the dir doesn't exist.

(Else `test` would return an error code)

(I'm assuming there's a `set -e` above)

We should remove once this is rolled out to all envs - but since it's not yet, I needed a test.

Reply:
Oh I thought the `test -d` was sufficient to make the `|| true` redundant. Ok thanks.